come-lavoriamo/consuntivazione.md

```diff
@@ -1,62 +1,89 @@
+lavorative spese sui diversi progetti e dall'altro delle ore di presenza in ufficio.
+
+### Le ore lavorative
+
+L'effort su un progetto viene individuato con un numero di ore, o più semplicemente di giornate,
+settimanali da spendere sul progetto stesso che é importante conoscere e tenere presente sempre. Se
+avete dubbi sulla quantità di ore da svolgere su un progetto a voi assegnato non esitate a chiederlo
+al vostro mentore.
+
+> Se salto una giornata su un progetto di un cliente devo [recuperare il venerdì?](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md#lavorare-sui-clienti-il-venerd%C3%AC)
+
+Il tempo svolto effettivamente sul progetto deve essere poi misurato. Adottiamo un time tracker per
+diversi motivi:
+
+- onestà e trasparenza;
+- efficenza amministrativa;
+- facilità di utilizzo;
+- precisione dei dati.
+
+É fondamentale tenere presente che l'attività che svolgiamo è soprattutto creativa. Non poniamoci
+degli obiettivi in termini di ore da svolgere e cerchiamo sempre di dare la precedenza alla qualità
+del lavoro. Lasciamo che il tracker sia un semplice strumento per la consuntivazione delle ore, non
+un misuratore della nostra produttività.
+
+Tuttavia, dobbiamo essere consapevoli delle sfide che il time tracking porta con sé.
+
+Una è sicuramente quella di definire quali sono i task da attribuire ad un progetto. Non c'è
+un'interpretazione univoca nel nostro settore. Tuttavia in linea con quelli che sono [i nostri valori](https://github.com/nebulab/playbook/blob/master/su-di-noi/valori.md)
+adottiamo la seguente soluzione. Sono, quindi, da includere:
+
+- tempo di prestazione diretta o implementazione;
+- tempo di lavoro in pairing sullo stesso progetto;
+- tempo di comunicazione direttamente correlato con il lavoro che si sta svolgendo;
+- tempo di studio direttamente ed esclusivamente correlato con il lavoro che si sta svolgendo.
+
+Si dovrebbe invece switchare il tracker su un altro progetto per:
+
+- tempo di prestazione diretta o implementazione su altri progetti;
+- tempo di lavoro in pairing con un'altra persona su altri progetti;
+- tempo di comunicazione non correlato con il lavoro che si sta svolgendo;
+- tempo di studio non direttamente e non esclusivamente correlato con il lavoro che si sta svolgendo.
+
+> Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici
 devono essere inclusi? La risposta è: non trovarsi mai nella situazione di doversi girare i pollici
 e trovare qualcosa da fare comunque. Se proprio non si ha nulla da fare questo tempo NON va incluso
 nel "tempo effettivamente svolto" ma ne va data pronta comunicazione al cliente, segnalando
 l'inefficienza.
 
-Per segnare su Float i tempi di pausa e i tempi di attesa utilizziamo il progetto `Time-Wasting`.
+Non serve far partire il tracker per i tempi di pausa ma lasciamo la libertà, per chi volesse, di
+segnare questi momenti con il progetto `Time wasting`.
```

Comment:
Can you use `**` (which will make the text bold) instead of the backticks, which are used for code?